### PR TITLE
Added `themes` and `layouts` to Button

### PR DESCRIPTION
**Pull request by peterp**
I've brought this inline with our current setup:

screen shot 2016-08-26 at 15 46 46
screen shot 2016-08-26 at 15 46 35
screen shot 2016-08-26 at 15 46 16

### CODE REVIEW

**Comment by peterp** [comment]
@justinvdm this is ready for review

File: src/components/Button/index.js [hunk 1]
```diff
@@ -41,12 +40,26 @@ const Button = ({
...
-  theme: PropTypes.oneOf([THEME_DEFAULT, THEME_WHITE]),
+  theme: PropTypes.oneOf([
+    themes.default,
+    themes.light,
+    themes.transparent,
+  ]),
+  layout: PropTypes.oneOf([
+    layouts.default,
+    layouts.inline,
+  ]),
```
Comment (justinvdm) [inline src/components/Button/index.js]: Wondering if we do in fact want to limit layouts and themes to ones provided by the component. Seems possible that we'd want to provide a special-cased theme or layout. Thoughts?

**Comment by justinvdm** [comment]
👍 !

File: src/components/Button/index.js (hunk 1 above)
Comment (peterp) [inline src/components/Button/index.js]: Yeah, agreed.

**Pull request merged**